<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QWidget" name="controlWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="0">
       <widget class="QLabel" name="hueLabel">
        <property name="text">
         <string>hue:</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="frequencyLabel">
        <property name="text">
         <string>frequency</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSlider" name="backlightCompensationSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="gammaLabel">
        <property name="text">
         <string>gamma</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1">
       <widget class="QPushButton" name="defaultBtn">
        <property name="text">
         <string>default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="brightnessSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="whiteBalanceComboBox"/>
      </item>
      <item row="15" column="1">
       <widget class="QSlider" name="gainSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSlider" name="gammaSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="contrastLabel">
        <property name="text">
         <string>contrast:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSlider" name="hueSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSlider" name="contrastSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSlider" name="saturationSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="brightLabel">
        <property name="text">
         <string>brightness:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="exposureLabel">
        <property name="text">
         <string>exposure</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="whiteBalanceLabel">
        <property name="text">
         <string>white balance:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QComboBox" name="exposureComboBox"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="saturationLabel">
        <property name="text">
         <string>saturation:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="sharpnessLabel">
        <property name="text">
         <string>sharpness:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="backlightCompensationLabel">
        <property name="text">
         <string>backlight compensation</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSlider" name="sharpnessSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QSlider" name="exposureAbsoluteSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSlider" name="whiteBalanceSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QComboBox" name="frequenceComboBox"/>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="gainLabel">
        <property name="text">
         <string>gain</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="exposureAbsoluteLabel">
        <property name="text">
         <string>exposure absolute</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
